Modules/_ssl.c

```diff
@@ -3914,8 +3914,8 @@ _ssl__SSLContext_load_cert_chain_impl(PySSLContext *self, PyObject *certfile,
             /* the password callback has already set the error information */
         }
         else if (errno != 0) {
-            ERR_clear_error();
             PyErr_SetFromErrno(PyExc_OSError);
+            ERR_clear_error();
```

Comment:
Is this function related to errno? Or is it specific to ssl and it leaves errno unchanged?

Reply:
Ok, I'm fine with this change.